src/content/guides/getting-started.md

```diff
@@ -71,6 +71,32 @@ __index.html__
 </html>
 ```
 
+We also need to adjust our `package.json` file in order to make sure we mark our package as `private`, as well as removing the `main` entry. This is to prevent an accidental publish of your code.
```

Comment:
I would honestly remove this paragraph, it isn't wrong but it feels out of place honestly. This is a quick start guide to webpack and I feel we are bombarding the user with tons of stuff that for a small project are not necessary.

Sorry if it sounds hard or agressive, it's not the intention. It's also hard to me to _garden_ the content that, though useful, doesn't belong to documentation, especially when it is aimed to people new at a tool or library.

Reply:
I would leave the tip quote below though. We are communicating that there's more to know around a topic and pointing the user to the right place to look for it.

Reply:
Yeah it was a suggestion by @ljharb . I agree that it should be as simple as we can make it. I can take it out for now (and leave the tip) and see if we get comments on it in the future. We could always add it in later.

Reply:
👍